### PR TITLE
Release v0.2.4

### PR DESCRIPTION
**Pull request by fubuloubu**
Depends: #2125

### CODE REVIEW

File: docs/release-notes.rst [hunk 1]
```diff
@@ -3,6 +3,23 @@
...
 #############
 
+v0.2.4
+******
+
+Date released: 03-08-2020
+
+Non-breaking changes and improvements:
+
+- Improve EOF Exceptions (`#2103 <https://github.com/vyperlang/vyper/pull/2115>`_)
```
Comment (iamdefinitelyahuman) [inline docs/release-notes.rst:13]:
```suggestion
- Improve EOF Exceptions (`#2115 <https://github.com/vyperlang/vyper/pull/2115>`_)
```

The numbers shown in the link text don't line up with the numbers in the actual links.

File: docs/release-notes.rst [hunk 2]
```diff
@@ -3,6 +3,23 @@
...
+- Improve exception messaging for type mismatches (`#2102 <https://github.com/vyperlang/vyper/pull/2119>`_)
```
Comment (fubuloubu) [inline docs/release-notes.rst:14]:
```suggestion
- Improve exception messaging for type mismatches (`#2119 <https://github.com/vyperlang/vyper/pull/2119>`_)
```

File: docs/release-notes.rst [hunk 3]
```diff
@@ -3,6 +3,23 @@
...
+- Ignore trailing newline tokens (`#2101 <https://github.com/vyperlang/vyper/pull/2120>`_)
```
Comment (fubuloubu) [inline docs/release-notes.rst:15]:
```suggestion
- Ignore trailing newline tokens (`#2120 <https://github.com/vyperlang/vyper/pull/2120>`_)
```

File: docs/release-notes.rst [hunk 4]
```diff
@@ -3,6 +3,23 @@
...
+
+Fixes:
+
+- Fix ABI translations for structs that are returned from functions (`#2110 <https://github.com/vyperlang/vyper/pull/2114>`_)
```
Comment (fubuloubu) [inline docs/release-notes.rst:19]:
```suggestion
- Fix ABI translations for structs that are returned from functions (`#2114 <https://github.com/vyperlang/vyper/pull/2114>`_)
```

File: docs/release-notes.rst [hunk 5]
```diff
@@ -3,6 +3,23 @@
...
+- Raise when items that are not types are called (`#2108 <https://github.com/vyperlang/vyper/pull/2118>`_)
```
Comment (fubuloubu) [inline docs/release-notes.rst:20]:
```suggestion
- Raise when items that are not types are called (`#2118 <https://github.com/vyperlang/vyper/pull/2118>`_)
```

File: docs/release-notes.rst [hunk 6]
```diff
@@ -3,6 +3,23 @@
...
+- Ensure hex and decimal AST nodes are serializable (`#2096 <https://github.com/vyperlang/vyper/pull/2123>`_)
```
Comment (fubuloubu) [inline docs/release-notes.rst:21]:
```suggestion
- Ensure hex and decimal AST nodes are serializable (`#2123 <https://github.com/vyperlang/vyper/pull/2123>`_)
```

**Pull request merged**